app/components/inputFields/SelectOptionTextField.tsx

```diff
@@ -0,0 +1,63 @@
+import { MenuItem, TextField } from "@mui/material";
+import React from "react";
+
+const roles = [
+  {
+    value: "student",
+    label: "Student",
+  },
+  {
+    value: "teacher",
+    label: "Teacher",
+  },
+];
+
+const SelectOptionTextField = () => {
+  return (
```

Comment:
take props and add TypeScript interface
```
interface SelectOptionTextFieldProps extends Omit<TextFieldProps, "variant"> {
  options?: typeof PROFESSION_OPTIONS;
}
```


```suggestion
const SelectOptionTextField: React.FC<SelectOptionTextFieldProps> = ({
  options = PROFESSION_OPTIONS,
  className = "w-[420px] mx-auto",
  ...props
}) => {
  return (
```

Reply:
done